app.py

```diff
@@ -99,12 +114,13 @@ class Organization(db.Model):
     last_updated = db.Column(db.Integer())
```

Comment:
@ondrae Is there any reason why the last_updated is an integer in Organizations and a DateTime in Projects?